dokka-runners/dokka-gradle-plugin/src/main/kotlin/tasks/DokkaGenerateTask.kt

```diff
@@ -114,7 +126,13 @@ constructor(
         generationType: GeneratorMode,
         outputDirectory: File,
     ) {
-        val dokkaConfiguration = createDokkaConfiguration(generationType, outputDirectory)
+        val dokkaConfiguration =
+            if (overrideJsonConfig.isPresent) {
+                logger.warn("w: [$path] Overriding DokkaConfiguration with manualJsonConfig")
+                DokkaConfigurationImpl(overrideJsonConfig.get())
```

Comment:
what's `manualJsonConfig `? I don't see it anywhere. Perhaps you meant `overrideJsonConfig ` :)